db/migrate/20250215085838_add_unique_index_name_and_deleted_at_to_tax_categories.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddUniqueIndexNameAndDeletedAtToTaxCategories < ActiveRecord::Migration[7.0]
+  def change
+    add_index(:spree_tax_rates, [:name, :deleted_at], unique: true,
+                                                      name: :index_spree_tax_rates_on_name)
```

Comment:
It for `spree_tax_categories` table :
```suggestion
    add_index(:spree_tax_categories, [:name, :deleted_at], unique: true,
                                                      name: :index_spree_tax_rates_on_name)

Reply:
```sql
SELECT name, deleted_at, COUNT(*) FROM spree_tax_categories GROUP By name, deleted_at HAVING COUNT(*) >1
```

Reply:
Oops. I will change this one.

Reply:
The production servers don't reveal any duplicates here. Good to go!

Reply:
The SQL query doesn't show any duplicates. :+1: